src/main/java/com/tstepnik/planner/domain/task/Task.java

```diff
@@ -0,0 +1,44 @@
+package com.tstepnik.planner.domain.task;
+
+import com.tstepnik.planner.domain.user.User;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import java.time.LocalDateTime;
+
+@Entity
+@Getter
+@Setter
+@NoArgsConstructor
+public class Task {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @NotEmpty
+    private String description;
+
+    private boolean isDone;
+
+    private boolean isArchived;
+
+    @Enumerated(EnumType.STRING)
+    private Importance importance;
+
+    @ManyToOne
+    private User user;
+
+    private LocalDateTime creationDate;
+    private LocalDateTime expirationDate;
```

Comment:
But expiration date is somethind different that planning date. I'm not sure if you understand my point. Every task have life time for the end of the day which was created but if we want to planned task for feature, then thier expiration time will be end of the day which was planned for.

Reply:
Yeah, that is exactly what I thought and your code doesn't work like that. Also because of what you just described I think we don't need expiration date field